### DISCUSSION TITLE
How to change style of scrollbar in listbox or listview in WPF?

### DISCUSSION THREAD

**Discussion by VahidEra** [dotnet/wpf#7800] [Q&A] [answered]
Hi guys,
I have two ResourceDictionary in my WPF project which include MaterialDesignInXamlToolkitand and HandyControl.

```
<ResourceDictionary  Source="pack://application:,,,/HandyControl;component/Themes/Theme.xaml"/>
<ResourceDictionary Source="pack://application:,,,/MaterialDesignExtensions;component/Themes/MaterialDesignLightTheme.xaml" />
```

And I want to use the scrollbar style of the listbox and listview of my project from the HandyControl component.(Only scrollbar style and i don't want to use scrollviewer)
Can I do this and if so how should I do it?
This is my Sample project:
https://drive.google.com/file/d/1pk6hEv29tF4SzUvyRTWjeJlFC4uNbpQe/view?usp=sharing
Thank you very much.

**Comment by miloush** [comment, marked as answer]
```xaml
    <ListBox.Resources>
        <Style TargetType="ScrollBar" BasedOn="{StaticResource ScrollBarBaseStyle}" />
    </ListBox.Resources>
```

should do the trick since neither of the themes explicitly sets style on scrollbars. It will affect all scrollbars inside the ListBox though. If you don't want that, you will have to override the control template.

**Reply by VahidEra** [reply]
@miloush Dear Jan, This is exactly what I was looking for, thank you very much.
You saved my time.
